lib/strategy.js

```diff
@@ -47,7 +47,8 @@ function Strategy(options, verify) {
   options.authorizationURL = options.authorizationURL || 'https://medium.com/m/oauth/authorize';
   options.tokenURL = options.tokenURL || 'https://api.medium.com/v1/tokens';
   options.scope = options.scope || ["basicProfile", "listPublications", "publishPost"]
-
+  options.state = options.state || "keyboardcat" 
```

Comment:
How about, instead of setting a default state, we throw an error. `state` also serves as a security purpose, so a publicly known state would be a really bad practice.

Reply:
@kristof0425 Do you think there are anymore changes that need to be made?

Reply:
@abdulhannanali No, throwing an error seems to be a good enough solution. 🙂 Chiming in @helloworld as he is the owner of this PR. Thanks for reviewing the PR, @abdulhannanali!